docs/content/preview/yugabyte-platform/manage-deployments/start-stop-processes.md

```diff
@@ -21,6 +21,8 @@ In some cases, the system might experience an unrecoverable error. To mitigate,
 
 If a node needs the intervention, you can click its associated **Actions > Stop Processes**.
 
-Once the TServer and, possibly, Master server are stopped, the node status is updated and the instance is ready for the planned system changes.
+Once the YB-TServer and, possibly, YB-Master server are stopped, the node status is updated and the instance is ready for the planned system changes.
 
-It is recommended not to stop more than (RF - 1)/2 processes at any given time. For example, on an RF=3 cluster with three nodes, there can only be one node with stopped processes to allow the majority of the nodes to perform Raft consensus operations.
+When a YB-Master is stopped on a node, YugabyteDB Anywhere automatically attempts to start a new YB-Master on another node in the same Availability Zone as the node on which YB-Master is stopped.  This ensures that the number of YB-Master servers equals the replication factor (RF) and YB-Master servers are never underreplicated.
+
+It is recommended not to stop more than (RF - 1) / 2 processes at any given time. For example, on an RF=3 cluster with three nodes, there can only be one node with stopped processes to allow the majority of the nodes to perform Raft consensus operations.
```

Comment:
This part is more nuanced than this. Maybe we can add a "In general, it is recommended" kind of a wording to avoid explaining the full nuance.

Reply:
Sure

Reply:
I added "Generally", but I'm not sure about.

Reply:
@iSignal Should I merge this PR?